doc/source/tutorial-series-get-started-with-flower-pytorch.ipynb

```diff
@@ -40,7 +42,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install -q flwr[simulation] flwr_datasets[vision] torch torchvision matplotlib"
+    "!pip install -q flwr-nightly[simulation] flwr-datasets[vision] torch torchvision matplotlib"
```

Comment:
I wonder why do we need flwr-nightly here.

Reply:
After 1.9 release, I'll remove these. 😄